R/LSgameofchance.R

```diff
@@ -169,27 +169,27 @@ LSgameofchance   <- function(jaspResults, dataset, options, state = NULL){
 
   
   ## Summary Table
-  summaryTable <- createJaspTable(title = "Summary Table")
+  summaryTable <- createJaspTable(title = gettext("Summary Table"))
```

Comment:
It's good to wrap all strings that are gonna be displayed in the output within `gettext` / `gettextf` so they can be translated later. (https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/translate.md)